pandas/_libs/tslibs/frequencies.pyx

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+import re
+
+cimport cython
+
+import numpy as np
+cimport numpy as np
+np.import_array()  # Without this, `is_integer_object` causes segfaults
+
+from util cimport is_integer_object
+
+
+@cython.ccall
```

Comment:
what does this decorator do?

Reply:
```
@cython.ccall
def foo(x):
```

is equivalent to `cpdef foo(x):`

Reply:
then use ``cpdef foo(x)``; don't try to change the existing style in a single PR.